app/constants/urls.js

```diff
@@ -0,0 +1,6 @@
+// need to get actual links to stores
+export const APPSTORE_URL = 'itms-apps://itunes.apple.com/us/app/id1501903733';
+export const PLAYSTORE_URL = 'market://details?id=edu.mit.privatekit';
+// placeholder json for testing, to be changed
+export const VERSION_URL =
+  'https://raw.githubusercontent.com/Fatkhi/covid-safe-paths/releases/versions.json';
```

Comment:
Can you add a separate PR for this JSON file to be included in the repository? so that we test here directly

Reply:
@penrods suggested that we should use the CDN via https://raw.githack.com/.  He's apparently talked with them about this use case.  Related ... might need to do the same for the healthcare-authority.yaml file.  Need him to verify.